parm/atm/berror/hybvar_gsibec.yaml.j2

```diff
@@ -61,6 +61,8 @@ components:
             level for 2d variables: last
           nicas:
             resolution: 6
+            min effective resolution: 0.2  # Note: this is very much not recommended (resolution too low), only
```

Comment:
We have a similar CI kludge in `parm/atm/lgetkf/lgetkf.yaml.j2`
```
local ensemble DA:
  solver: GETKF
  vertical localization:
#   fraction of retained variance: 0.975  # use for scientific tests
    fraction of retained variance: 0.750  # use for ctest
```
My preference is to use the _science_ value in both `lgetkf.yaml.j2` and `hybvar_gsibec.yaml.j2`.   I'm hopeful we can do so as we process a more representative set of observations in these tests. 

We could use variables in both cases.  There are other things which we may also want to _variablize_ (e.g, EnVar weights, observation errors, etc).  I'm reluctant to open this can of worms at present.

What do you think?

Reply:
What are the consequences of using the 'science' values in the CI tests? Computation time? I agree we don't want to open that can of worms yet, but I worry that if the default value in the repo is not ideal, it will fall through the cracks and users will have degraded analyses in their testing.

Reply:
When the minimum effective resolution is not specified in the g-w JEDI ATM case, the 2024020400 gdas and gfs atmanlrun abort with the message.
```
 0: Info     :                 Compute horizontal subset Sc1:
 0: Info     :                    Decimate full grid, at least    13824 points required,   13824 valid points found
 0: Info     :                    Potential method: 0% 5% 10% 15% 20% 25% 30% 35% 40% 45% 50% 55% 60% 65% 70% 75% 80% 85% 90% 95% 100%
 0: Info     :                    Subgrid hash:  -1651493416
 0: Info     :                 Final nc1:    13824
 0: Info     :                 Effective horizontal resolution:  2.83
65: !!! ABORT in nicas_cmp_compute_horizontal on task #000066: effective resolution is too low, please increase resolution and/or max horizontal grid size\
 parameters, or decrease min effective resolution (risky)
```
Given the message in the log, I looked at yamls in `fv3-jedi/test/testinput`.   I found `min effective resolution: 0.2` and dropped it into `hybvar_gsibec.yaml.j2`.  

JEDI ATM g-w CI is using a two member ensemble to keep the computational cost low.  A larger ensemble might remove the need for `min effective resolution 0.2`.

Reply:
Gotcha. Ok, let's merge for now, and remember to do either a variable or a Jinja IF block "soon"

Reply:
Thank you @CoryMartin-NOAA